pipeline/ingestor/src/main/scala/uk/ac/wellcome/platform/ingestor/config/builders/IngestorConfigBuilder.scala

```diff
@@ -23,4 +24,23 @@ object IngestorConfigBuilder {
       index = Index(indexName)
     )
   }
+
+  def intFromConfig(config: Config,
+                    path: String,
+                    default: Option[Int] = None): Int =
+    Try(config.getAnyRef(path))
+      .map {
+        case value: String  => value.toInt
+        case value: Integer => value.asInstanceOf[Int]
+        case obj =>
+          throw new RuntimeException(
+            s"$path is invalid type: got $obj (type ${obj.getClass}), expected Int")
+      }
+      .recover {
+        case exc: ConfigException.Missing =>
+          default getOrElse {
+            throw new RuntimeException(s"${path} not defined in Config")
+          }
+      }
+      .get
```

Comment:
🎸

Reply:
Shamelessly copied. Maybe that should be extracted somewhere common but for now, I'm going to merge as is